src/inc/eventtracebase.h

```diff
@@ -426,7 +523,7 @@ namespace ETW
         
         static VOID DomainLoad(BaseDomain *pDomain, __in_opt LPWSTR wszFriendlyName = NULL)
         {
-            if (MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER_Context.IsEnabled)
+            if (ETW_PROVIDER_ENABLED(MICROSOFT_WINDOWS_DOTNETRUNTIME_PROVIDER))
```

Comment:
Sounds good.  Can you file an issue to change the implementation of ETW_PROVIDER_ENABLED and the other IsEnabled event methods once LTTng supports them?